sys/crypto/tweetnacl/tweetnacl.c

```diff
@@ -0,0 +1,810 @@
+#include "tweetnacl.h"
+#define FOR(i,n) for (i = 0;i < n;++i)
+#define sv static void
```

Comment:
I would vote to replace such defines with their original keywords,
even if it grow larger than the original 100 tweets NaCl implementation